distributions/aws/examples/cognito/README.md

```diff
@@ -12,7 +12,8 @@ This guide assumes that you have:
     - [kubectl](https://kubernetes.io/docs/tasks/tools) - A command line tool for working with Kubernetes clusters.
     - [yq](https://mikefarah.gitbook.io/yq) - A command line tool for YAML processing. (For Linux environments, use the [wget plain binary installation](https://mikefarah.gitbook.io/yq/#wget))
     - [jq](https://stedolan.github.io/jq/download/) - A command line tool for processing JSON.
-    - [kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) - A command line tool to customize Kubernetes objects through a kustomization file.
+    - [kustomize version 3.2.0](https://github.com/kubernetes-sigs/kustomize/releases/tag/v3.2.0) - A command line tool to customize Kubernetes objects through a kustomization file.
```

Comment:
Do we know if we are actually able to support kustomize 3.2.x instead of just 3.2.0 ?

Reply:
haven't tried other versions, I grabbed this guidance from the main Kubeflow README https://github.com/awslabs/kubeflow-manifests#prerequisites.